### ISSUE TITLE
GoogleNet with Cross Entropy Benchmark

### ISSUE THREAD

**Issue by cenkbircanoglu** [zalandoresearch/fashion-mnist#33]
I use GoogleNet for feature extraction with Cross Entropy Loss function and embedding creation part, and I use Linear SVM for classifier.
I got accuracies as;
Train: 0.9980
Test: 0.9365

**Comment by philip-bl** [comment]
Do you know by any chance how many parameters your GoogleNet had? Or was it just the standard GoogleNet which I can download off the internet?
